ts-rs/tests/issue-70.rs

```diff
@@ -30,15 +32,25 @@ fn issue_70() {
     );
 }
 
+type GenericAlias<A = String, B = String> = HashMap<(A, String), Vec<(B, i32)>>;
+
+#[derive(TS)]
+#[ts(export, export_to = "tests-out/issue_70/")]
+struct Container {
+    a: GenericAlias<Vec<i32>, Vec<String>>,
```

Comment:
This is gonna be a problem as `Record<K, V>` requires that `K extends string | number | symbol`